templates/summary_manager/handle_experience.html.twig

```diff
@@ -1,45 +1,76 @@
-{% extends 'summary_manager/layout.html.twig' %}
-
-{% block summary_content %}
-<section class="l__wrapper-500">
-
-    <h2 class="b__nudge--bottom">Éditer l'expérience</h2>
-
-    <div class="cv__form">
-        {{ form_start(experience_form) }}
-
-            {{ form_errors(experience_form) }}
-
-            {{ form_row(experience_form.company, { label: 'Organisation' }) }}
-            {{ form_row(experience_form.position, { label: 'Poste' }) }}
-            {{ form_row(experience_form.location, { label: 'Ville' }) }}
-            {{ form_row(experience_form.website, { label: 'Site internet de l\'organisation' }) }}
-            {{ form_row(experience_form.company_facebook_page, { label: 'Page Facebook de l\'organisation' }) }}
-            {{ form_row(experience_form.company_twitter_nickname, { label: 'Compte Twitter de l\'organisation' }) }}
-            {{ form_row(experience_form.started_at, { label: 'Début' }) }}
-            {{ form_row(experience_form.ended_at, { label: 'Fin' }) }}
-            {{ form_row(experience_form.on_going, { label: 'J\'y travaille actuellement' }) }}
-            {{ form_row(experience_form.contract, { label: 'Type de contrat' }) }}
-            {{ form_row(experience_form.duration, { label: 'Statut' }) }}
-            {{ form_row(experience_form.description, { label: 'Description du poste' }) }}
-
-            {% if experience_form.display_order.entry.vars.choices is defined %}
-                {{ form_row(experience_form.display_order, { label: 'Position dans la liste de vos expériences' }) }}
-            {% else %}
-                {{ form_widget(experience_form.display_order) }}
-            {% endif %}
-
-        {{ form_end(experience_form) }}
-
-        {% if experience_form.vars.data and not experience_form.vars.data.new %}
-            {{ delete_form(path('app_summary_manager_remove_experience', { id: experience_form.vars.data.id }), constant('AppBundle\\Summary\\SummaryManager::DELETE_EXPERIENCE_TOKEN')) }}
-        {% endif %}
-        <div class="b__nudge--bottom">
-            <a href="{{ path('app_summary_manager_index') }}">
-                Retourner à mon profil En Marche
-            </a>
+{% extends 'summary_manager/_form_layout.html.twig' %}
+
+{% block summary_form %}
+    <h2 class="b__nudge--bottom">Ajouter / Modifier une expérience</h2>
+
+    {{ form_start(experience_form) }}
+
+    {{ form_errors(experience_form) }}
```

Comment:
(minor comment) You should indent between `form_start` and `form_end` for consistency.